### PR TITLE
switched to committed stream type

### PR DESCRIPTION
**Pull request by raahi-chada2004**

### CODE REVIEW

File: out_writeapi.go [hunk 1]
```diff
@@ -262,6 +263,12 @@ func FLBPluginInit(plugin unsafe.Pointer) int {
 	datasetID := output.FLBPluginConfigKey(plugin, "DatasetID")
 	tableID := output.FLBPluginConfigKey(plugin, "TableID")
 
+	//set exactly-once bool from config file param
+	exactlyOnceVal, err := strconv.ParseBool(output.FLBPluginConfigKey(plugin, "Exactly_Once"))
```
Comment (agrawal-siddharth) [inline out_writeapi.go]: If the parameter is absent we should default to False.

**Review by agrawal-siddharth — CHANGES_REQUESTED** [review]
Please also update the README file to document the Exactly_Once parameter.

File: out_writeapi.go (hunk 1 above)
Comment (raahi-chada2004) [reply out_writeapi.go]: Done.

File: README.md [hunk 2]
```diff
@@ -70,9 +70,10 @@ Here is an example of an `OUTPUT` section:
...
+The `Max_Chunk_Size` field takes in the number of bytes that the plugin will attempt to chunk data into before appending into the BigQuery Table. Fluent-Bit supports around a maximum of 2 MB of data within a single flush and we exercise a hard maximum of 9 MB (as BigQuery cannot handle appending data larger than this size). The `Max_Queue_Requests` and `Max_Queue_Bytes` fields describe the maximum number of requests/bytes of outstanding asynchrous responses that can be queued. When the first limit is reached, data appending will be blocked until enough responses are ready and the number of outstanding requests/bytes decreases. The `Exactly_Once` field takes in a boolean that describes whether exactly-once semantics will be utilized. With exactly-once delivery, response checking will be synchronous (as opposed to asynchronous response checking with at-least once/default semantics).
```
Comment (agrawal-siddharth) [inline README.md]: Not sure why there isn't a line break before the description of Exactly_Once. You can also mention that by default this is turned off.

**Review by agrawal-siddharth — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (raahi-chada2004) [reply README.md]: Done.

**Review by agrawal-siddharth — APPROVED** [review]

**Pull request merged**